hyp3_gamma/insar/etc/README_InSAR_GAMMA.txt

```diff
@@ -144,6 +145,15 @@ The digital elevation file contains the terrain height for the scenes being proc
 
 *Note that this file is optional. Select the "Include DEM" option in the HyP3 Processing Options to include it in the product package.*
 
+----------------
+## 11. Water Mask
+
+The water mask specifies, pixel-by-pixel, where large waterbodies exist in the corresponding InSAR product. Pixel values of 0 indicate areas that should be masked out, while areas with a pixel value of 1 are included in the InSAR processing. This mask is applied to the DEM prior to running InSAR jobs, effectively clipping out the water, as areas with DEM heights of 0 are considered invalid for InSAR processing. The water mask is stored as a byte-valued GeoTIFF file and is included in the product package when water masking is applied.
```

Comment:
> as areas with DEM heights of 0 are considered invalid for InSAR processing.

I don't think this is true anymore, right?

Reply:
I changed the words to reflect how the water mask is applied to the proecess.